seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckConfigUtil.java

```diff
@@ -26,17 +26,14 @@
 
 public class CheckConfigUtil {
 
-    public static CheckResult check(Config config, String... params) {
-        StringBuilder missingParams = new StringBuilder();
-        for (String param : params) {
-            if (!config.hasPath(param) || config.getAnyRef(param) == null) {
-                missingParams.append(param).append(",");
-            }
-        }
+    public static CheckResult checkAllExists(Config config, String... params) {
```

Comment:
1. since it's in common module, should we keep `check()` method and add an `@Deprecated` annotation to keep API compatible?
2. the new method named `requireAllExist` be better.

Reply:
+1

Reply:
make sense, thanks.